docs/api-docs/management-apis/contextual-filters.md

```diff
@@ -0,0 +1,310 @@
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+> * Your store plan must support Product Filtering to use contextual filters. 
+> * Product Filtering is available for Pro and Enterprise plans. To upgrade, see [Changing Your Store's Plan](https://support.bigcommerce.com/s/article/How-do-I-upgrade-my-stores-plan).
+> * Enterprise accounts have the ability to use [custom fields](https://support.bigcommerce.com/s/article/Custom-Fields) as product filters to provide shoppers with additional ways to customize their search. 
+
+</div>
+</div>
+</div>
+
+## View pre-configured filters
+
+There are two ways to retrieve the global filters pre-configured on your store: using the Settings API or through the store's control panel. To retrieve all pre-configured global filters using the API, send a `GET` request to [/v3/settings/search/filters](insert-link-here).
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#requestrunner) 
+// replace link
+
+The response will contain all of the global filters currently configured on your store. 
+
+```json
+{
+   "data": [
+       {
+           "id": "Y2F0ZWdvcnk=",
+           "display_name": "Category",
+           "type": "category",
+           "items_to_show": 15,
+           "collapsed_by_default": false,
+           "display_product_count": true,
+           "is_enabled": true
+       },
+       {
+           "id": "U2l6ZQ==",
+           "display_name": "Size",
+           "type": "product",
+           "display_product_count": true,
+           "collapsed_by_default": false,
+           "items_to_show": 10,
+           "sort_by": "alpha",
+           "is_enabled": true
+       },
+       {
+           "id": "cHJpY2U=",
+           "display_name": "Price",
+           "type": "price",
+           "collapsed_by_default": false,
+           "is_enabled": true
+       }
+       ...
+   ],
+   "meta": {}
+}
+```
+The global filters returned from completing a `GET` request should match the filters you see in the control panel under **Storefront Filters**. This set of filters is what you currently have configured on your store. 
+
+To access **Storefront Filters** in the control panel, go to **Products** > **Product Filtering**. 
+
+![Products Filters](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/contextual-filters-01.png "Products Filters")
+
+## View available filters
+
+Available filters are the filters you can configure on the global level. To see all available configurable filters, send a `GET` request to [`/v3/settings/search/filters/available`](insert-link-here). To retrieve a list of filters available on a given category, add `category_id` as a query parameter.
```

Comment:
This contradicts itself here, because the first sentence says "this is something at the global level", but later on it says "here's how to get it on a category level", which is not the global level.

Perhaps a better sequence is:

1. here's a thing that gives you the available filters in a certain context
2. here's how to get them for the global context
3. here's how to get it for the category context on channel 1
4. (later) here's how to get it for other contexts

Reply:
I think we can also mention here that the list of available filters is pulled dynamically based on the setup of the catalog. You "create" new available filters by changing your product information in your catalog, such as adding new product options or custom fields. I think it's fine to link to merchant KB for the details of that.